lib/transaction.js

```diff
@@ -39,15 +39,13 @@ function mixinTransaction(DB2, db2) {
       if (err) return cb(err);
       connection.beginTransaction(function(err) {
         if (isolationLevel) {
-          var sql = 'SET CURRENT ISOLATION TO ' + isolationLevel;
+          var stmt = {};
+          stmt.sql = 'SET CURRENT ISOLATION TO ' + isolationLevel;
+          stmt.noResults = true;
 
-          if (sql) {
-            connection.query(sql, function(err) {
-              cb(err, connection);
-            });
-          } else {
+          connection.query(stmt, function(err) {
             cb(err, connection);
-          }
+          });
```

Comment:
Are these changes relevant to implementation of replaceById? Shouldn't they be in a separate commit with appropriate commit message?